core/core-impl/src/test/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/list/ListChangeMoveSelectorFactoryTest.java

```diff
@@ -118,14 +104,14 @@ void explicitConfigMustUseListVariable() {
         ListChangeMoveSelectorConfig config = new ListChangeMoveSelectorConfig()
                 .withValueSelectorConfig(new ValueSelectorConfig("value"))
                 .withDestinationSelectorConfig(new DestinationSelectorConfig()
-                        .withEntitySelectorConfig(new EntitySelectorConfig(TestdataMixedVariablesEntity.class))
+                        .withEntitySelectorConfig(new EntitySelectorConfig(TestdataEntity.class))
```

Comment:
The Entity above mixes basic + list vars. For this test case, use the basic one.